src/VisualStudio/Core/Def/Implementation/ProjectSystem/MetadataReferences/VisualStudioMetadataReference.Snapshot.cs

```diff
@@ -82,15 +82,7 @@ private bool SaveMetadataReadingException(Exception e)
 
             protected override DocumentationProvider CreateDocumentationProvider()
             {
-                string xmlDocumentPath;
-                if (ReferencePathUtilities.TryFindXmlDocumentationFile(FilePath, out xmlDocumentPath))
```

Comment:
looks like this helper is a linked file (workspaces.desktop and scripting(portable)) and scripting uses it for [MetadataShadowCopying](https://github.com/dotnet/roslyn/blob/4b0669ac892800b041c49bdd4bb3336112b7f912/src/Scripting/Core/AssemblyLoader/MetadataShadowCopyProvider.cs#L453).

But sure, I can clean up a bit (remove code that scripting doesn't need) and move it to to scripting from workspaces.